exporters/scss/src/js/normalizers/weight.ts

```diff
@@ -1,4 +1,10 @@
-export function normalizeWeight(name: string): number | string {
+export function normalizeWeight(name: string, fontFamily: string): number | string {
+  // Ebony has a font weight named "Semibold" that is actually 400 weight, we do this
+  // exception here to match Adobe Fonts output
+  if (fontFamily === 'Ebony' && name.toLowerCase().replace(/\W/g, '') === 'semibold') {
```

Comment:
I suggest refactoring this and creating the map instead of `if` and `switch`. See following:

```js
const defaultWeights = {
  thin: 100,
  extralight: 200,
  // ...
}

const normalizedWeights = {
  default: defaultWeights,
  ebony: {
    ...defaultWeights,
    semibold: 400,
  }
}
```

Then set the fontFamily argument to have `default`.

```js
return normalizedWeights[fontFamily.toLowerCase()][name.toLowerCase().replace(/\W/g, '')];
```

Reply:
Also, the map can be in a separate file between constants or something. It will be much easier to extend this and much more performant.

Reply:
Had to add types as well 😨 Anyway updated, check it please